### ISSUE TITLE
Test Case: Verify Clinical Trial Search Results functionality

### ISSUE THREAD

**Issue by SupriyaSaxena0605** [NCIOCPL/cgov-digital-platform-qa#156]
https://www.cancer.gov/about-cancer/treatment/clinical-trials/search/r

# Test Scenarios:
- [x] 1. Verify that the Search Criteria dropdown box is displayed on clicking the drop-down in the Search Results page if at least one search parameter has been entered.
- [x] 2. Verify that selecting the 'Select All' checkbox, all the checkboxes gets selected.
- [x] 3. Verify when all trials on a page is selected, the "Select all on Page" checkbox gets selected.
- [x] 4.Verify that deselecting one of the checkbox, deselects the 'Select All' checkbox. 
- [x] 5. Verify that on clicking a trial user is directed to respective Trial Description page url.
- [x] 6. Verify that when "Modify your search" is clicked and user is redirected to Advanced CTS form (www.cancer.gov/about-cancer/treatment/clinical-trials/search/advanced) the search parameters are retained on the Advanced CTS page.

below is covered in the separate issue (#157 )
**- [] 7. Verify when one trial is selected and 'Print Selected' button is clicked, the  user is redirected to the Print Page https://www.cancer.gov/CTS.Print/Display?printid=xxxxx**
**- [] 8. Verify when "Select all on page" is selected and 'Print Selected' button is clicked, the  user is redirected to the Print Page https://www.cancer.gov/CTS.Print/Display?printid=xxxxx**

- [x] 9. Verify the default count of records on the page.
- [x] 10. Verify that the pagination displays the 'Previous' and 'Next' Links.
- [x] 11. Verify when 'Previous' link is clicked, user is redirected to the Previous page.
- [x] 12. Verify when 'Next' link is clicked, user is redirected to the Next page.
- [x] 13. Verify when user is on the first page, the 'Previous' link is not displayed
- [x] 14. Verify when user is on the last page, the 'Next' link is not displayed
- [x] 15.Verify the valid page number is displayed on the current page the user is on.
- [x] 16. Verify when Page 2 is selected from the Pager, the Results count header displays the text "Results 11-20"

Depends on Issue #<number>
------------------------------------
Review comments:
 5.Verify that 'Category' and 'Your Selection' headers are displayed in the Search Criteria box. - move to Search Criteria ticket.....moved to https://github.com/NCIOCPL/cgov-digital-platform-qa/issues/100
- write a test for checking if 'Start Over' and 'Modify Search Criteria' links are present......covered in https://github.com/NCIOCPL/cgov-digital-platform-qa/issues/93

- write test for validating correct text on modals.....covered on https://github.com/NCIOCPL/cgov-digital-platform-qa/issues/168

- for Negative tests, both are the same tests, just copy and paste. And they are not actually negative tests, as you have tests for modals in positive tests too.........removed the duplicate test
- write tests for pagination also......added test cases above
- write tests for checking appearance of select Al checkbox and Print Selection twice on the page.......covered on covered in https://github.com/NCIOCPL/cgov-digital-platform-qa/issues/93

- [x] 12. Verify when no trial is selected and "Print Selected" button is clicked, a modal appears. - this needs to be more specific, as if when no trial is selected, the modal will have an error message displayed "You have not selected any trials. Please select at least one trial to print.".......Please see https://github.com/NCIOCPL/cgov-digital-platform-qa/issues/168

**Comment by kate-mashkina** [comment]
PR #172 has closed this issue
(https://github.com/NCIOCPL/cgov-digital-platform-qa/pull/172)
